libs/pmp-web/repository/shell/src/lib/shell-routing.module.ts

```diff
@@ -2,6 +2,13 @@ import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
 
 const routes: Routes = [
+  {
+    path: 'projects',
```

Comment:
This is quite different than https://github.com/valueadd-poland/pimp-my-pr/pull/55/files#diff-48b19dd6af2e14d5d07ebcfec20d1780R8